lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -191,50 +398,6 @@ def _probabilities_to_percentiles(
         percentile_cube.cell_methods = {}
         return percentile_cube
 
-    def _get_bounds_of_distribution(self, forecast_probabilities):
```

Comment:
Moved to lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_utilities.py with minor change.